src/components/BulkEnrollmentPage/index.test.jsx

```diff
@@ -1,94 +1,138 @@
 import { Provider } from 'react-redux';
+import userEvent from '@testing-library/user-event';
 import { renderWithRouter } from '../test/testUtils';
 import BulkEnrollmentPage from './index';
 import '../../../__mocks__/react-instantsearch-dom';
 import LicenseManagerApiService from '../../data/services/LicenseManagerAPIService';
 import { ROUTE_NAMES } from '../EnterpriseApp/constants';
 import { ADD_COURSES_TITLE } from './stepper/constants';
+import '@testing-library/jest-dom';
 
 jest.mock('../../data/services/LicenseManagerAPIService', () => ({
   __esModule: true,
   default: {
     fetchSubscriptions: jest.fn(),
+    fetchSubscriptionUsersOverview: jest.fn(),
   },
 }));
 
+const testSlug = 'horsesRus';
+
 const sub1 = {
   uuid: 'foo',
   title: 'horse subscription',
-  enterpriseSlug: 'horsesRus',
+  enterpriseSlug: testSlug,
   startDate: 1621347682,
   expirationDate: 1721547682,
   licenses: {
     allocated: 5,
     total: 20,
   },
+  enterpriseCatalogUuid: 'blarghl2',
 };
 
 const sub2 = {
   uuid: 'bearz',
   title: 'bear subscription',
-  enterpriseSlug: 'horsesRus',
+  enterpriseSlug: testSlug,
   startDate: 1621347682,
   expirationDate: 1721547682,
   licenses: {
     allocated: 15,
     total: 40,
   },
+  enterpriseCatalogUuid: 'blarghl',
 };
 
+const mockSubscriptionCount = Promise.resolve({
+  data: [
+    { status: 'unassigned', count: 0 },
+    { status: 'assigned', count: 0 },
+    { status: 'activated', count: 2 }],
+});
 const subscriptions = Promise.resolve({
   data: {
-    results: {
-      subscriptions: [sub1, sub2],
-    },
+    results: [sub1, sub2],
     errors: null,
     setErrors: jest.fn(),
     forceRefresh: jest.fn(),
     loading: false,
+    count: 2,
   },
 });
-
-LicenseManagerApiService.fetchSubscriptions.mockReturnValue(subscriptions);
+const singleSubscription = Promise.resolve({
+  data: {
+    results: [sub1],
+    errors: null,
+    setErrors: jest.fn(),
+    forceRefresh: jest.fn(),
+    loading: false,
+    count: 1,
+  },
+});
+const mockSingleSubscriptionCount = Promise.resolve({
+  data: [
+    { status: 'unassigned', count: 0 },
+    { status: 'assigned', count: 0 },
+    { status: 'activated', count: 1 }],
+});
 
 const fakeStore = {
   portalConfiguration: {
     enterpriseId: 'foo',
+    enableCodeManagementScreen: false,
+    enableSubscriptionManagementScreen: true,
   },
 };
 const mockStore = configureMockStore([thunk]);
 const BulkEnrollmentWrapper = () => (
-  <MemoryRouter>
-    <Provider store={mockStore(fakeStore)}>
-      <BulkEnrollmentPage />
-    </Provider>
-  </MemoryRouter>
+  <Provider store={mockStore(fakeStore)}>
+    <BulkEnrollmentPage />
+  </Provider>
 );
 
-describe('<BulkEnrollmentPage />', () => {
+LicenseManagerApiService.fetchSubscriptions.mockImplementation(() => subscriptions);
+LicenseManagerApiService.fetchSubscriptionUsersOverview.mockImplementation(() => mockSubscriptionCount);
+
+describe('<BulkEnrollmentPage /> Multiple Subscriptions', () => {
```

Comment:
Nit: I would do
```
describe('<BulkEnrollmentPage />', () => {
  describe('multiple subscriptions', () => {
  })
})
```
So that it's all the same test suite.